tests/integration/utils/test_sqlalchemy_query_utils.py

```diff
@@ -32,5 +32,7 @@ def test_insert_many(engine):
         connection.execute(insert_many)
         response = connection.execute(sqlalchemy.select(table))
         results = list(response)
+        # Explicitly drop the table as it persists in the Trino engine
```

Comment:
Oh yeah, I guess it must persist in the MSSQL engine too (it's just a vanilla table, not a temporary one) but for whatever reason it doesn't cause us problems there. Cleaning up like this seems sensible in any case.